spl-ids/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "solana-spl-ids"
+description = "Solana spl ids"
+documentation = "https://docs.rs/solana-spl-ids"
+publish = false
```

Comment:
nit: This will need to be published, right?

Reply:
Sorry for missing this!

Reply:
umm. does it?

Reply:
I don't think we'll be able to use `solana-program-test` in other projects if it isn't published :sweat_smile:

Reply:
ah true